src/what-does-rust-run-on.md

```diff
@@ -0,0 +1,39 @@
+# What does Rust run on?
+
+---
+
+## Host vs Target
+
+* The machine you develop on
+* The machine the program runs on
+
+---
+
+## Rust is a cross-compiler
+
+* It uses LLVM to generate machine code
+* *Every* Rust install is a cross-compiler
+  * No rummaging for extra installers for your platform
```

Comment:
I would make an emphasis (or speaker's note) here on the fact that Rust can enter any part of your stack and help you build fast, safe, reliable tooling and applications for all production OS's, so it's a great efficiency multiplier across an organization when cross-platform concerns are minimized.